### PR TITLE
Removed commands considered to be useless.

### PR DESCRIPTION
**Pull request by Shillelagh-Hummus**
## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by Qustinnus** [comment]
what even is this pr lol

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -5,19 +5,19 @@ GLOBAL_PROTECT(admin_verbs_default)
 /world/proc/AVerbsDefault()
 	return list(
 	/client/proc/deadmin,				/*destroys our own admin datum so we can play as a regular player*/
-	/client/proc/cmd_admin_say,			/*admin-only ooc chat*/
+	//client/proc/cmd_admin_say,			/*admin-only ooc chat*/
```
Comment (Qustinnus) [inline code/modules/admin/admin_verbs.dm:8]: if you're going to remove things actually remove them, dont comment them out

**Comment by Qustinnus** [comment]
ah okay i see this is bait

File: code/modules/admin/admin_verbs.dm (hunk 1 above)
Comment (Ashes2007) [reply code/modules/admin/admin_verbs.dm:8]: I accidentally requested a PR on the wrong github lol. I meant to do it with my own fork and pushed to all.

**Pull request closed without merging**